### DISCUSSION TITLE
Capsule docs

### DISCUSSION THREAD

**Discussion by michaeljcoyne** [area17/twill#1878] [General]
Hi I am trying to install the redirects capsule on the show and tell.

But I don't know what Capsules are as there are no docs I can find on this.

Where is the best place to start to understand capsules?

**Comment by ifox** [comment]
Hi @michaeljcoyne, capsules are modules encapsulated in a single folder. They do need documentation, but there isn't much to them beyond copying them in your project and enabling them in your configuration. Please share the error if you need more help.

**Reply by michaeljcoyne** [reply]
Hi @ifox 
So I have followed the instructions.

https://github.com/area17/twill-capsule-redirections/issues/1

But obviosuly the code is installed in a sub directory with migrations to run etc. I have no idea what the process is to get this working. 

You say you just need to enable them in the configuration but I get the error the tables from the migrations in the capsule are not there. 

I am not sure if there is a helper for this I don't know about.

Thanks

Michael

**Reply by ifox** [reply]
Hi @michaeljcoyne, did you run `php artisan migrate` after adding it to the configuration?

**Reply by michaeljcoyne** [reply]
Yes it is working now. I tried it before but it said nothing to migrated but I had my capsuels in the dashboard array in config and not the root. Working now thanks.
